src/EventHub/EventHub/help/Set-AzEventHub.md

```diff
@@ -45,10 +45,10 @@ PS C:\> $createdEventHub.CaptureDescription.Destination.Name = "EventHubArchive.
 PS C:\> $createdEventHub.CaptureDescription.Destination.BlobContainer = "container"
 PS C:\> $createdEventHub.CaptureDescription.Destination.ArchiveNameFormat = "{Namespace}/{EventHub}/{PartitionId}/{Year}/{Month}/{Day}/{Hour}/{Minute}/{Second}"
 PS C:\> $createdEventHub.CaptureDescription.Destination.StorageAccountResourceId = "/subscriptions/{SubscriptionId}/resourceGroups/MyResourceGroupName/providers/Microsoft.ClassicStorage/storageAccounts/arjunteststorage"
-PS C:\> Set-AzEventHub -ResourceGroupName MyResourceGroupName -Namespace MyNamespaceName -Name MyEventHubName -InputObject MyCreatedEventHub -messageRetentionInDays 4 -partitionCount 2
```

Comment:
Why do you remove the "-messageRetentionInDays 4 -partitionCount 2"

Reply:
in order to change the 'Capture description' properties for an existing  **MyEventHubName** .
the proposal is to use minimum options.  if you add these two values.  
you keep getting error : - 
[error]Parameter set cannot be resolved using the specified named parameters.